components/IconButton/index.tsx

```diff
@@ -1,28 +1,37 @@
 import { Button, useMantineColorScheme } from "@mantine/core";
 import { LinkProps } from "next/link";
+import React, { MouseEventHandler } from "react";
 
 import style from "./_index.module.scss";
 
 type Props = Omit<LinkProps, "href"> & {
     icon: React.ReactNode;
     label: string;
-    className?: "link" | "active";
+    className?: "link" | "active" | "back";
+    onClick?: MouseEventHandler<HTMLButtonElement>;
```

Comment:
@jdesolate LinkProps has already onClick with it

Reply:
resolved